React/Base/RCTConvert.m

```diff
@@ -740,21 +740,21 @@ + (type)type:(id)json                                       \
   return colorMap;
 }
 
-/** Returns an UIColor based on a semantic color name.
+/** Returns a UIColor based on a semantic color name.
  *  Returns nil if the semantic color name is invalid.
  */
-static UIColor *RCTColorFromSemanticColorName(NSString *semanticColorName)
+static RCTUIColor *RCTColorFromSemanticColorName(NSString *semanticColorName)
```

Comment:
Missing the TODO comment

Reply:
This change is already inside a ` // [TODO(macOS ISS#2323203)` ... ` // ]TODO(macOS ISS#2323203)` block.